src/site/stages/build/plugins/add-asset-hashes.js

```diff
@@ -1,69 +1,85 @@
-function addAssetHashes(isPreview = false) {
-  // In non-development modes, we add hashes to the names of asset files in order to support
-  // cache busting. That is done via WebPack, but WebPack doesn't know anything about our HTML
-  // files, so we have to replace the references to those files in HTML and CSS files after the
-  // rest of the build has completed. This is done by reading in a manifest file created by
-  // WebPack that maps the original file names to their hashed versions. Metalsmith actions
-  // are passed a list of files that are included in the build. Those files are not yet written
-  // to disk, but the contents are held in memory.
+/* eslint-disable no-continue, no-param-reassign */
 
-  return (files, metalsmith, done) => {
-    // Read in the data from the manifest file.
-    const manifestKey = Object.keys(files).find(
-      filename => filename.match(/file-manifest.json$/) !== null,
-    );
+const environments = require('../../../constants/environments');
 
-    const originalManifest = JSON.parse(files[manifestKey].contents.toString());
+const FILE_MANIFEST_FILENAME = 'generated/file-manifest.json';
+
+function copyAssetsToTeamSitePaths(buildOptions, files, entryNamesDictionary) {
+  // TeamSite pages such as "benefits.va.gov" have hardcoded references to certain
+  // JavaScript bundles so that we can inject our header/footer into pages outside
+  // of our source. Therefore, we can't apply a hash to those bundles.
+  const teamsiteInjectionBundles = {
+    'proxy-rewrite.js': 'generated/proxy-rewrite.entry.js',
+    'vendor.js': 'generated/vendor.entry.js',
+    'polyfills.js': 'generated/polyfills.entry.js',
+    'styleConsolidated.css': 'generated/styleConsolidated.css',
+    'static-pages.css': 'generated/static-pages.css',
+  };
```

Comment:
This may be overkill, but it would eliminate some redundancy:

```js
  const teamsiteFiles = [
    'proxy-rewrite.js',
    'vendor.js',
    'polyfills.js',
    'styleConsolidated.css',
    'static-pages.css'
  ]

  const teamsiteInjectionBundles = teamsiteFiles.reduce((obj, fileName) => { 
    const [ name, ext] = fileName.split('.')
    obj[fileName] = `generated/${name}.entry.${ext}`
    return obj
    }, {})
```

Reply:
I personally prefer the explicitness of the hardcoded string values, because this is kind of an unclear operation for an edge of the whole build.

Reply:
Also, `.css` doesn't get the `.entry` piece applied 🤷‍♂